common/smoketest/requirements-release.txt

```diff
@@ -0,0 +1,8 @@
+azure-core>=0.0.0b1
```

Comment:
Are we certain this will automatically pick the latest pre-release version? I'm not an expert but if I'm reading https://www.python.org/dev/peps/pep-0440/#handling-of-pre-releases correctly it will exclude pre-releases for these version identifiers unless they are already installed.

Reply:
It does, as I think appending `b1` causes this to happen. For example, as of this writing `azure-storage-blob` has versions `12.6.0` and `12.7.0b1` published upstream. By setting the version to `>= 0.0.0b1`, pip installs `12.7.0b1`, with no cache:

```
⇉ ⇉ ⇉ python -m venv ./prereleasetest
⇉ ⇉ ⇉ cd prereleasetest && . ./bin/activate
⇉ ⇉ ⇉ cd ~/sdk/azure-sdk-for-python/common/smoketest
⇉ ⇉ ⇉ pip3 install -r requirements-release.txt --no-deps --upgrade  --no-cache-dir | grep storage
Collecting azure-storage-blob>=0.0.0b1
  Downloading azure_storage_blob-12.7.0b1-py2.py3-none-any.whl (332 kB)
...
⇉ ⇉ ⇉ pip3 freeze | grep storage
azure-storage-blob==12.7.0b1
```

Reply:
@scbedd might have more context on the underlying behavior, as he showed me this tip originally.

Reply:
If it works then great.